dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -85,7 +88,8 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::RegisterL
         return S_OK;
     }
 
-    m_notificationListenerManager.AddListener(appId, processName);
+    AddComServerClsid(appId, comServerClsid);
```

Comment:
So for unpackaged apps that rely on Protocol, comServerCLSID is GUID_NULL correct?